lib/index.js

```diff
@@ -24,6 +24,12 @@ GA.ensure(function(msg, settings){
   return this.reject('Must set either server-side or mobile tracking ID');
 });
 
+GA.ensure(function(msg, _){
+  if ('screen' != msg.type()) return;
```

Comment:
does `.type()` return the message type? i thought `action()` did.

ie. 

```
if (msg.action() !== 'screen') return;
```

also lets add a comment above specifying why we're doing this :)